.github/workflows/cwf-integ-test.yml

```diff
@@ -12,14 +12,16 @@ on:  # yamllint disable-line rule:truthy
       - v1.*
     types:
       - completed
+env:
+  SHA: ${{ github.event.workflow_run.head_commit.id || github.sha }}
 
 jobs:
   docker-build:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
         with:
-          ref: ${{ github.event.workflow_run.head_commit.id || github.sha }}
+          ref: ${{ env.sha }}
```

Comment:
is env context case sensitive? why not use env.SHA just like we do everywhere else?

Reply:
It's not case sensitive but better use CAPS everywhere